### DISCUSSION TITLE
Multiple instances of input-remapper-service in system processes

### DISCUSSION THREAD

**Discussion by a-e-egorov** [sezanzeb/input-remapper#584] [General]
Hello! I noticed 4(!) similar processes in my system monitor (OS Ubuntu 20) called "input-remapper-service" - see the attached screenshot from my system monitor. They all start at system start. I'm just curious - is it normal; or it's kind of bug somewhere, which multiplies program launches, and normally only one process must exist?

![Снимок экрана от 2022-12-25 18-11-17](https://user-images.githubusercontent.com/18039483/209473388-27b67419-7b66-4712-acd1-18586ec9b2a4.png)

**Comment by jonasBoss** [comment]
That is normal, there are different parts running in different processes.
